src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -470,7 +470,7 @@ def test_credscache_add_new_sp_creds(self, _, mock_open_for_write, mock_read_fil
             test_sp2['servicePrincipalTenant'])
 
         # assert
-        token_entries = [entry for _, entry in creds_cache.adal_token_cache.read_items()]
+        token_entries = [e for _, e in creds_cache.adal_token_cache.read_items()]  # noqa: F812
```

Comment:
Question: What's 'noqa: F812'?
Is there a non-code version like how for PyLint we have 'line-too-long'?
That way, we can see what the comment is for without having to know what F812 means.